2024-11-14 23:36:51.627 [info] Experiment 'pythonRecommendTensorboardExt' is active
2024-11-14 23:36:51.627 [info] Experiment 'pythonTerminalEnvVarActivation' is active
2024-11-14 23:36:51.627 [info] Experiment 'pythonTestAdapter' is active
2024-11-14 23:36:51.627 [warning] Dir "/home/yoshee/hse/machine_learning/.pixi/envs" is not watchable (directory does not exist)
2024-11-14 23:36:51.672 [info] Default formatter is set to esbenp.prettier-vscode for workspace /home/yoshee/hse/machine_learning
2024-11-14 23:36:51.691 [info] Starting Environment refresh
2024-11-14 23:36:51.691 [info] Searching for interpreters in posix paths locator
2024-11-14 23:36:51.692 [info] Searching for pyenv environments
2024-11-14 23:36:51.692 [info] Searching for conda environments
2024-11-14 23:36:51.692 [info] Searching for global virtual environments
2024-11-14 23:36:51.692 [info] Searching for custom virtual environments
2024-11-14 23:36:51.706 [info] pyenv is not installed
2024-11-14 23:36:51.707 [info] Finished searching for pyenv environments: 14 milliseconds
2024-11-14 23:36:51.708 [info] Finished searching for custom virtual envs: 16 milliseconds
2024-11-14 23:36:51.716 [info] > conda info --json
2024-11-14 23:36:51.725 [info] > pixi --version
2024-11-14 23:36:51.732 [info] > hatch env show --json
2024-11-14 23:36:51.732 [info] cwd: .
2024-11-14 23:36:51.733 [info] Python interpreter path: /bin/python
2024-11-14 23:36:51.745 [info] Finished searching for global virtual envs: 53 milliseconds
2024-11-14 23:36:52.071 [info] > pyenv which python
2024-11-14 23:36:52.071 [info] cwd: .
2024-11-14 23:36:52.106 [info] Found: /bin/python --> /bin/python3.12
2024-11-14 23:36:52.118 [info] Found: /bin/python3 --> /bin/python3.12
2024-11-14 23:36:52.118 [info] Found: /bin/python3.12 --> /bin/python3.12
2024-11-14 23:36:52.120 [warning] could not find a pixi interpreter for the interpreter at /bin/python
2024-11-14 23:36:52.154 [warning] could not find a pixi interpreter for the interpreter at /bin/python
2024-11-14 23:36:52.167 [info] > /bin/python -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.196 [info] Found: /sbin/python --> /sbin/python3.12
2024-11-14 23:36:52.210 [info] Found: /sbin/python3 --> /sbin/python3.12
2024-11-14 23:36:52.210 [info] Found: /sbin/python3.12 --> /sbin/python3.12
2024-11-14 23:36:52.233 [info] Found: /usr/bin/python --> /usr/bin/python3.12
2024-11-14 23:36:52.234 [info] Found: /usr/bin/python3 --> /usr/bin/python3.12
2024-11-14 23:36:52.234 [info] Found: /usr/bin/python3.12 --> /usr/bin/python3.12
2024-11-14 23:36:52.278 [info] Found: /usr/sbin/python --> /usr/sbin/python3.12
2024-11-14 23:36:52.278 [info] Found: /usr/sbin/python3 --> /usr/sbin/python3.12
2024-11-14 23:36:52.278 [info] Found: /usr/sbin/python3.12 --> /usr/sbin/python3.12
2024-11-14 23:36:52.279 [info] Found: /home/linuxbrew/.linuxbrew/bin/python3 --> /home/linuxbrew/.linuxbrew/Cellar/python@3.12/3.12.6/bin/python3.12
2024-11-14 23:36:52.279 [info] Found: /home/linuxbrew/.linuxbrew/bin/python3.12 --> /home/linuxbrew/.linuxbrew/Cellar/python@3.12/3.12.6/bin/python3.12
2024-11-14 23:36:52.293 [info] > /sbin/python -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.306 [info] > /usr/bin/python -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.313 [info] Finished searching for interpreters in posix paths locator: 626 milliseconds
2024-11-14 23:36:52.315 [info] Environments refresh paths discovered (event): 629 milliseconds
2024-11-14 23:36:52.315 [info] Environments refresh paths discovered: 629 milliseconds
2024-11-14 23:36:52.341 [info] > /usr/sbin/python -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.354 [info] > /home/linuxbrew/.linuxbrew/bin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.429 [info] Environments refresh finished (event): 743 milliseconds
2024-11-14 23:36:52.431 [info] Environment refresh took 745 milliseconds
2024-11-14 23:36:52.433 [error] Environment /bin/python got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.434 [error] Environment /usr/sbin/python got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.434 [error] Environment /sbin/python got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.434 [error] Environment /usr/bin/python got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.466 [info] > /sbin/python3.12 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.471 [info] > /usr/sbin/python3.12 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.503 [info] > /bin/python3.12 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.504 [error] Environment /sbin/python3.12 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.511 [info] > /usr/bin/python3.12 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.512 [error] Environment /usr/sbin/python3.12 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.540 [info] > /usr/sbin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.541 [error] Environment /bin/python3.12 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.552 [info] > /usr/bin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.553 [error] Environment /usr/bin/python3.12 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.578 [info] > /bin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.579 [error] Environment /usr/sbin/python3 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.590 [info] > /sbin/python3 -I ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/python_files/interpreterInfo.py
2024-11-14 23:36:52.591 [error] Environment /usr/bin/python3 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.612 [error] Environment /bin/python3 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.628 [error] Environment /sbin/python3 got the wrong version from native locator (Native = 3.12.6.final.0, Actual 3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910])
2024-11-14 23:36:52.629 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
2024-11-14 23:36:52.643 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
2024-11-14 23:36:53.178 [info] Starting Pylance language server.
